tapir/coop/models.py

```diff
@@ -20,7 +20,7 @@
 
 
 class ShareOwner(models.Model):
-    """ShareOwner represents an owner of a ShareOwnership.
+    """ShareOwner represents an share_owner of a ShareOwnership.
```

Comment:
Here "owner" is the word owner more than the variable name, I would leave it as just "owner"